tests.dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM 996097627176.dkr.ecr.us-east-1.amazonaws.com/data-zapier-nightly:latest
+
+ENV ENVIRONMENT=production
+
+COPY bootstrap.R /tmp/bootstrap.R
+COPY DESCRIPTION /tmp/DESCRIPTION
+WORKDIR /tmp
+
+RUN Rscript bootstrap.R
+
+ADD . /code
+WORKDIR /code
+
+#RUN Rscript ci.R
+
+RUN R CMD build .
```

Comment:
... or is that happening here?

Reply:
Nowhere actually, this is just building `tar.gz`. `R CMD check` is still failing on `zapieR`, `database.R`

Reply:
I know we talked about this a bit before, but I didn't really get it then either.  I think what you are saying is that after you build redshiftTools then zapieR's CMD CHECK fails, but that if you zapieR's CMD CHECK alone (or with an older version of redshiftTools) it passes?

Reply:
Actually it seems that as part of the `redshiftTools` CMD check the dependancies are checked as well, so the `redshiftTools` check fails. Steps to reproduce:

+ R CMD BUILD .
+ R CMD CHECK <tarball>